src/auxinfo/proof.rs

```diff
@@ -13,8 +13,8 @@ use crate::{
     participant::InnerProtocolParticipant,
     ring_pedersen::VerifiedRingPedersen,
     zkp::{
-        pifac::{CommonInput, PiFacProof, ProverSecret},
-        pimod::{PiModInput, PiModProof, PiModSecret},
+        pifac::{CommonInput as PiFacCommonInput, PiFacProof, ProverSecret as PiFacProverSecret},
+        pimod::{CommonInput as PiModCommonInput, PiModProof, ProverSecret as PiModProverSecret},
```

Comment:
@gatoWololo do you have an aesthetic preference between this vs using the module path, like `pifac::ProverSecret`? I have a slight preference for the latter.

Reply:
I also prefer the module path. Otherwise, it feels like we are introducing more names to keep track of.